src/js/msp/MSPHelper.js

```diff
@@ -1392,6 +1392,13 @@ MspHelper.prototype.process_data = function(dataHandler) {
                 FC.BLACKBOX.blackboxMode = data.readU8();
                 FC.BLACKBOX.blackboxDenom = data.readU16();
                 FC.BLACKBOX.blackboxFields = data.readU32();
+                if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_12_8)) {
+                    FC.BLACKBOX.blackboxInitialEraseKiB = data.readU16();
+                    FC.BLACKBOX.blackboxRollingErase = data.readU8();
+                } else {
+                    FC.BLACKBOX.blackboxInitialEraseKiB = 0;
+                    FC.BLACKBOX.blackboxRollingErase = 0;
```

Comment:
Not required. FC state gets reset elsewhere.

Reply:
Removed